src/components/ListItem.vue

```diff
@@ -0,0 +1,114 @@
+<template>
```

Comment:
This component mostly duplicates the functionality of `SettingsItem`, so I propose to rename `SettingsItem` to `ListItem` and use it instead with `AeRadio` component like in [SettingsNetwork component](https://github.com/aeternity/aepp-identity/blob/6d94730/src/pages/SettingsNetwork.vue).